python/ray/_raylet.pxd

```diff
@@ -101,7 +101,8 @@ cdef class CoreWorker:
     cdef _create_put_buffer(self, shared_ptr[CBuffer] &metadata,
                             size_t data_size, ObjectRef object_ref,
                             c_vector[CObjectID] contained_ids,
-                            CObjectID *c_object_id, shared_ptr[CBuffer] *data)
+                            CObjectID *c_object_id, shared_ptr[CBuffer] *data,
+                            owner_address=*)
```

Comment:
What does this `=*` mean?

Reply:
It's how you declare optional arguments in Cython `.pxd` header files: https://cython.readthedocs.io/en/stable/src/userguide/language_basics.html#optional-arguments